e2e/utils.go

```diff
@@ -1265,3 +1269,12 @@ func validateSubvolumegroup(f *framework.Framework, subvolgrp string) error {
 	}
 	return nil
 }
+
+func upgradeCSI(version string, testtype string) {
```

Comment:
you may need to refer to https://github.com/ceph/ceph-csi/blob/master/e2e/rbd.go#L39-L124 for deployment of ceph-csi for upgrade testing

Reply:
Thanks for the inputs, I'll get back with the changes.